serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_oc.adoc

```diff
@@ -0,0 +1,8 @@
+[[proc-deploy-sw-application-openfhift-client]]
+=== Deploying your workflow application using `oc`
+
+
+
+// verify deployed swf
```

Comment:
Do we need all these spaces between the title and the include?

Reply:
no, removed :)